### ISSUE TITLE
dbt run processes csv files in data-paths with seed config set to enabled: false

### ISSUE THREAD

**Issue by vaughnkoch** [fishtown-analytics/dbt#794]
Hi, if you have a large csv file in data-paths, dbt will somehow process it despite it being excluded.
This makes dbt hang even though you don't want to process the large file.

`dbt -d run`:

```
2018-06-13 18:10:12,971: Parsing [redacted]/data/large_data_file.csv
```

Relevant config:
```
seeds:
  my_project:
    enabled: false
    schema: seed_data

    large_data_file:
      enabled: false
```

Full config:
[dbt_project.yml.txt](https://github.com/fishtown-analytics/dbt/files/2100438/dbt_project.yml.txt)

Data dir:
```
$ tree data
data
└── large_data_file.csv  # 63MB.
```
I'd suggest that the seeds config be required so that people know explicitly what's being processed. Making processing all seeds the default makes things work automagically, but can be confusing later on.  Also, any csv files should only be parsed when running dbt seed, or at least there should be an option to not run dbt seed automatically with dbt run.

Thanks for creating a great tool btw :)

**Comment by drewbanin** [comment]
Thanks for the report @vaughnkoch. dbt _does_ need to "find" the CSV files, as they can be `ref()`d from models, and it needs to know if the `ref` is valid or not. We definitely don't need to be loading the data up into memory though.

**Comment by drewbanin** [comment]
This has been fixed (#1568) for some time!
